apps/script/log-build-times.js

```diff
@@ -77,6 +83,7 @@ function uploadLoggedStatsToNewRelic(consoleLog) {
     consoleLog("Sending " + dataToLog.length + " build time events to new relic ");
     var failed = false;
     dataToLog.forEach(function (data) {
+      newrelic.agent.config.no_immediate_harvest = true;
```

Comment:
This looks like something we could/should set as soon as we load the agent, instead of here inside a `forEach` loop.